code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -465,7 +446,7 @@
 		if((heat_capacity > 0) && (abs(delta_temperature) > MINIMUM_TEMPERATURE_DELTA_TO_CONSIDER))
 
 			var/heat = thermal_conductivity*delta_temperature* \
-				(heat_capacity*700000/(heat_capacity+700000)) //700000 is the heat_capacity from a space turf, hardcoded here
+				(heat_capacity*HEAT_CAPACITY_VACUUM/(heat_capacity+HEAT_CAPACITY_VACUUM))
```

Comment:
technically `HEAT_CAPACITY_VACUUM` is 7000 but since these were hardcoded my guess is that it just never got updated when it needed to. calling this one a fix